### PR TITLE
Add component Departure

### PR DESCRIPTION
**Pull request by lxndrMr**

### CODE REVIEW

**Review by FabienRollet — APPROVED** [review]

**Review by pjroggy — APPROVED** [review]

File: client/src/components/Departure.jsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { useState } from "react"
+
+
+export default function Departure() {
+
+    const[selectedDeparture, SetSelectedDeparture] = useState('')
```
Comment (Yoann-Fortin) [inline client/src/components/Departure.jsx]: Are you sure ? Not use a class or a functionnal component to a function

File: client/src/components/Departure.jsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    
+    const handleDeparture = (e) => {
+        SetSelectedDeparture(e.target.value)
```
Comment (Yoann-Fortin) [inline client/src/components/Departure.jsx]: same

File: client/src/components/Departure.jsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    }
+    return (
+        <>
+            <label htmlFor="Departure">Arrivée à :</label>
```
Comment (Yoann-Fortin) [inline client/src/components/Departure.jsx]: missing id in input and no uppercase in the start of your value

File: client/src/components/Departure.jsx [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+            <input
```
Comment (Yoann-Fortin) [inline client/src/components/Departure.jsx]: missing attribut id to link with the correct label

**Review by Yoann-Fortin — CHANGES_REQUESTED** [review]

**Review by pjroggy — APPROVED** [review]

**Review by Yoann-Fortin — APPROVED** [review]

**Pull request merged**